### PR TITLE
fdtools: init at 2020.05.04

### PR DESCRIPTION
**Pull request by Profpatsch**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/fdtools/default.nix [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+let
+  pname = "fdtools";
+  version = "2020.05.04";
+  sha256 = "0lnafcp4yipi0dl8gh33zjs8wlpz0mim8mwmiz9s49id0b0fmlla";
+
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://code.dogmap.org/pname/releases/fdtools-${version}.tar.bz2";
```
Comment (Ma27) [inline pkgs/tools/misc/fdtools/default.nix:12]: should be `${pname}`, right?

**Comment by Profpatsch** [comment]
fixed

**Comment by Ma27** [comment]
@Profpatsch I'm afraid you misunderstood my comment. The URL is currently `code.dogmap.org/pname/releases/...` which is why I'm getting a 404 when trying to fetch the sources.

**Comment by Profpatsch** [comment]
LOL, fixed

**Pull request merged**